capabilities/recovering-capabilities.md

```diff
@@ -12,11 +12,13 @@ This most straightforward use of `recover` is to get an `iso` that you can pass
 
 ## What does this look like?
 
+The `recover` expression wraps a list of expressions and is terminated by an `end`, like this:
+
 ```pony
 recover Array[String] end
```

Comment:
we could also use `recover Array[String].create() end` here, if that is more clear.

Reply:
True. Is `Array[String]` considered more idiomatic? Then the tutorial should perhaps use it exclusively.

Reply:
If you write a lot of pony you learn to value the different syntactic sugars, so in this regard i would consider it more idiomatic. On the other side, if this is confusing when teaching about what can be done with `recover`, i think we should use the more explicit variant in this case.

Reply:
@mfelsche Fair enough, I have updated the pull request.